packages/terra-button/tests/wdio/button-spec.js

```diff
@@ -237,4 +237,24 @@ Terra.describeViewports('Button', ['tiny'], () => {
 
     Terra.it.validatesElement('Button is Disabled', { selector: '#root' });
   });
+
+  describe('Programmatically Re-enable a disabled button', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-button/button/programmatic-disable-enable');
+    });
+
+    it('disables button', () => {
+      browser.keys(['Tab']);
+      browser.click('#programmaticDisabledButton1');
+    });
+
+    Terra.it.validatesElement('Button is disabled', { selector: '#root' });
+
+    it('enables button', () => {
+      browser.keys(['Tab']);
+      browser.click('#programmaticDisabledButton2');
+    });
+
+    Terra.it.validatesElement('Enabled Button is not focused', { selector: '#root' });
```

Comment:
`Terra.it.validatesElement` is being deprecated. It will be replaced by `Terra.validates.element`.

```suggestion
    it('disables button', () => {
      browser.keys(['Tab']);
      browser.click('#programmaticDisabledButton1');
      Terra.validates.element('Button is disabled', { selector: '#root' });
    });

    it('enables button', () => {
      browser.keys(['Tab']);
      browser.click('#programmaticDisabledButton2');
      Terra.validates.element('Enabled Button is not focused', { selector: '#root' });
    });
```